pkg/ingresscontroller/ingresscontroller.go

```diff
@@ -0,0 +1,139 @@
+package ingresscontroller
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	v1 "github.com/openshift/api/operator/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ingresscontrollerName      = "default"
+	ingresscontrollerNamespase = "openshift-ingress-operator"
```

Comment:
Please remove this constant. It should be parameter. You can have this constant in test code